### PR TITLE
Update Swift API usage in Sasquatch apps

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [ ] ~Has `CHANGELOG.md` been updated?~
* [ ] ~Are tests passing locally?~
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update sample apps to reflect made changes in modules.

## Related PRs or issues

[AB#83344](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/83344)

### CODE REVIEW

File: Sasquatch/Sasquatch/MSAAnalyticsAuthenticationProvider.swift [hunk 1]
```diff
@@ -32,7 +32,7 @@ class MSAAnalyticsAuthenticationProvider: NSObject, MSACAnalyticsAuthenticationP
     }
     
     // Implement required method of MSACAnalyticsAuthenticationProviderDelegate protocol.
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/MSAAnalyticsAuthenticationProvider.swift:34]: fix comment pls

File: Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift [hunk 2]
```diff
@@ -122,7 +122,7 @@ class MSAnalyticsViewController: UITableViewController, AppCenterProtocol {
       for targetToken in MSTransmissionTargets.shared.transmissionTargets.keys {
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift:122]: MSTransmissionTargets?

File: Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift [hunk 3]
```diff
@@ -170,10 +170,10 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
   @IBAction func logFilterSwitchChanged(_ sender: UISwitch) {
     #if ACTIVE_COMPILATION_CONDITION_PUPPET
     if !eventFilterStarted {
-      MSACAppCenter.startService(MSEventFilter.self)
+      AppCenter.startService(MSEventFilter.self)
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift:173]: MSEventFilter?

File: Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift [hunk 4]
```diff
@@ -139,7 +139,7 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     self.pushEnabledSwitch.isOn = appCenter.isPushEnabled()
 
     #if ACTIVE_COMPILATION_CONDITION_PUPPET
-    self.logFilterSwitch.isOn = MSEventFilter.isEnabled()
+    self.logFilterSwitch.isOn = MSEventFilter.enabled
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift:142]: MSEventFilter?

File: Sasquatch/Sasquatch/ViewControllers/MSSignInViewController.swift [hunk 5]
```diff
@@ -76,8 +76,8 @@ class MSSignInViewController: UIViewController, WKNavigationDelegate {
             UserDefaults.standard.set(refreshToken, forKey: kMSARefreshTokenKey)
             
             // Create a MSACAnalyticsAuthenticationProvider and register as an MSACAnalyticsAuthenticationProvider.
-            let provider = MSACAnalyticsAuthenticationProvider(authenticationType: .msaCompact, ticketKey: userId, delegate: MSAAnalyticsAuthenticationProvider.getInstance(refreshToken, self))
-            MSACAnalyticsTransmissionTarget.addAuthenticationProvider(authenticationProvider:provider)
+            let provider = AnalyticsAuthenticationProvider(authenticationType: .msaCompact, ticketKey: userId, delegate: MSAAnalyticsAuthenticationProvider.getInstance(refreshToken, self))
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/ViewControllers/MSSignInViewController.swift:79]: MSAAnalyticsAuthenticationProvider?

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 6]
```diff
@@ -135,17 +135,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, MSACCrashesDelegate, UNUs
 
       // Add a "Don't send"-Button and call the notifyWithUserConfirmation-callback with MSACUserConfirmationDontSend
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:136]: naming in comment

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 7]
```diff
@@ -334,12 +334,12 @@ extension AppDelegate: MSACDistributeDelegate {
 
       // Add a "Yes"-Button and call the notifyUpdateAction-callback with MSUserAction.update
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:335]:
```suggestion
      // Add a "Yes"-Button and call the notifyUpdateAction-callback with .update
```

File: Sasquatch/SasquatchSwift/AppDelegate.swift [hunk 8]
```diff
@@ -334,12 +334,12 @@ extension AppDelegate: MSACDistributeDelegate {
...
       alertController.addAction(UIAlertAction(title: NSLocalizedString("distribute_alert_yes", tableName: "Sasquatch", comment: ""), style: .cancel) { _ in
-        MSACDistribute.notify(.update)
+        Distribute.notify(.update)
       })
 
       // Add a "No"-Button and call the notifyUpdateAction-callback with MSUserAction.postpone
```
Comment (DmitriyKirakosyan) [inline Sasquatch/SasquatchSwift/AppDelegate.swift:340]:
```suggestion
      // Add a "No"-Button and call the notifyUpdateAction-callback with .postpone
```

File: SasquatchMac/SasquatchMac/ViewControllers/AuthenticationViewController.swift [hunk 9]
```diff
@@ -99,8 +99,8 @@ class AuthenticationViewController: NSViewController, WKNavigationDelegate, MSAC
                         NSLog("Successfully signed in with user_id: %@", newUrl.valueOf("user_id")!)
 
                         // Create a MSACAnalyticsAuthenticationProvider and register as an MSACAnalyticsAuthenticationProvider.
```
Comment (DmitriyKirakosyan) [inline SasquatchMac/SasquatchMac/ViewControllers/AuthenticationViewController.swift:101]:
```suggestion
                        // Create a AnalyticsAuthenticationProvider and register as an AnalyticsAuthenticationProvider.
```

File: SasquatchMac/SasquatchMac/ViewControllers/AuthenticationViewController.swift [hunk 10]
```diff
@@ -129,7 +129,7 @@ class AuthenticationViewController: NSViewController, WKNavigationDelegate, MSAC
     }
 
     // Implement required method of MSACAnalyticsAuthenticationProviderDelegate protocol.
```
Comment (DmitriyKirakosyan) [inline SasquatchMac/SasquatchMac/ViewControllers/AuthenticationViewController.swift:131]:
```suggestion
    // Implement required method of AnalyticsAuthenticationProviderDelegate protocol.
```

File: SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift [hunk 11]
```diff
@@ -11,9 +11,9 @@ import AppCenterCrashes;
 class AppCenterDelegateSwift : AppCenterDelegate {
 
   // MARK: MSACAppCenter section.
```
Comment (DmitriyKirakosyan) [inline SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift:13]:
```suggestion
  // MARK: AppCenter section.
```

File: SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift [hunk 12]
```diff
@@ -24,17 +24,17 @@ class AppCenterDelegateSwift : AppCenterDelegate {
     // return MSACAppCenter.sharedInstance().logUrl()
```
Comment (DmitriyKirakosyan) [inline SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift:24]:
```suggestion
    // return AppCenter.sharedInstance().logUrl()
```

File: SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift [hunk 13]
```diff
@@ -50,97 +50,97 @@ class AppCenterDelegateSwift : AppCenterDelegate {
   // MARK: MSACCrashes section.
```
Comment (DmitriyKirakosyan) [inline SasquatchTV/SasquatchTVSwift/AppCenterDelegateSwift.swift:50]:
```suggestion
  // MARK: Crashes section.
```

File: Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift (hunk 3 above)
Comment (AnastasiaKubova) [reply Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift:173]: This is the class of our test app, not SDK

File: Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift (hunk 2 above)
Comment (AnastasiaKubova) [reply Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift:122]: This is the class of our test app, not SDK

File: Sasquatch/Sasquatch/ViewControllers/MSSignInViewController.swift (hunk 5 above)
Comment (AnastasiaKubova) [reply Sasquatch/Sasquatch/ViewControllers/MSSignInViewController.swift:79]: Same

File: Sasquatch/SasquatchSwift/AppCenterDelegateSwift.swift [hunk 14]
```diff
@@ -27,21 +27,21 @@ import AppCenterPush
...
 
-  func setCustomProperties(_ customProperties: MSACCustomProperties) {
-    MSACAppCenter.setCustomProperties(customProperties)
+  func setCustomProperties(_ customProperties: CustomProperties) {
+    AppCenter.setCustomProperties(customProperties)
   }
 
   func installId() -> String {
-    return MSACAppCenter.installId().uuidString
+    return AppCenter.installId().uuidString
```
Comment (MatkovIvan) [inline Sasquatch/SasquatchSwift/AppCenterDelegateSwift.swift:44]: Should this be an property too?

**Review by russelarms — APPROVED** [review]
Looks good to me.

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**